demo/configs/admin/content-type/release.json

```diff
@@ -66,7 +66,7 @@
                                 },
                                 "mappingOptions": {
                                     "index": null,
-                                    "analyzer": "keyword",
+                                    "analyzer": "version",
```

Comment:
Nice fix: it was not possible on "old" elasticsearch ..... introduce in around 7.10 (but missing in 7.6.0)

Reply:
Applied see next 2 commits, for now I will already merge :)